packages/js/components/src/tree-select-control/index.js

```diff
@@ -291,7 +292,11 @@ const TreeSelectControl = ( {
 					return acc;
 				}
 			} else if ( isSearching ) {
-				const match = option.label.toLowerCase().indexOf( filter );
+				const labelWithAccentsRemoved = removeAccents( option.label );
```

Comment:
would using [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) work here? If it does I think it's the more general option and doesn't introduce a dependency

Reply:
>would using [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) work here? If it does I think it's the more general option and doesn't introduce a dependency
Good call on not introducing a dependency. I managed to find another solution as `localeCompare` is more about determining string position when sorting.